### PR TITLE
Update doc copyrights to 2022

### PR DESCRIPTION
**Pull request by malfet**
Also, s/Torch/PyTorch/

### CODE REVIEW

**Comment by malfet** [comment]
@b0noI can we please make sure to track it with the releases?

**Review by kit1980 — APPROVED** [review]

**Review by soumith — APPROVED** [review]

File: docs/cpp/source/conf.py [hunk 1]
```diff
@@ -119,7 +119,7 @@
 
 # General information about the project.
 project = 'PyTorch'
-copyright = '2019, Torch Contributors'
+copyright = '2022, Torch Contributors'
```
Comment (malfet) [inline docs/cpp/source/conf.py:122]:
```suggestion
copyright = '2022, PyTorch Contributors'
```

File: docs/cpp/source/conf.py [hunk 2]
```diff
@@ -119,7 +119,7 @@
...
 author = 'Torch Contributors'
```
Comment (malfet) [inline docs/cpp/source/conf.py:123]:
```suggestion
author = 'PyTorch Contributors'
```

File: docs/source/conf.py [hunk 3]
```diff
@@ -261,7 +261,7 @@
 
 # General information about the project.
 project = 'PyTorch'
-copyright = '2019, Torch Contributors'
+copyright = '2022, Torch Contributors'
```
Comment (malfet) [inline docs/source/conf.py:264]:
```suggestion
copyright = '2022, PyTorch Contributors'
```

File: docs/source/conf.py [hunk 4]
```diff
@@ -261,7 +261,7 @@
...
 author = 'Torch Contributors'
```
Comment (malfet) [inline docs/source/conf.py:265]:
```suggestion
author = 'PyTorch Contributors'
```

**Pull request closed without merging**